qemu/tests/cfg/check_cvq_event.cfg

```diff
@@ -0,0 +1,7 @@
+- check_cvq_event:
+    type = check_cvq_event
+    clone_master = yes
+    master_images_clone = image1
+    remove_image_image1 = yes
+    change_cmd = "ip link set addr %s dev %s"
```

Comment:
So this case is `only Linux`

Reply:
> So this case is `only Linux`

Yes, you're right. Do I need to limit it in the cfg file?

Reply:
Limit this only test on Linux, so please help review it again, thanks a lot.